app/scripts/controller-init/messengers/seedless-onboarding/seedless-onboarding-controller-messenger.ts

```diff
@@ -36,13 +37,14 @@ export function getSeedlessOnboardingControllerMessenger(
   return messenger.getRestricted({
     name: 'SeedlessOnboardingController',
     allowedActions: [],
-    allowedEvents: ['KeyringController:lock', 'KeyringController:unlock'],
+    allowedEvents: [],
```

Comment:
Why do we no longer need those? In which cases does the controller get locked now?